charts/ingress-nginx/tests/controller-deployment_test.yaml

```
@@ -0,0 +1,33 @@
+suite: test controller deployment
+templates:
+  - controller-deployment.yaml
+
+tests:
+  - it: should always create a deployment
+    asserts:
+      - hasDocuments:
+          count: 1
+      - isKind:
+          of: Deployment
+
+  - it: custom replica count
+    set:
+      controller.enabled: true
```

```suggestion
      controller.enabled: true
```

There's no such value neither in the `values.yaml` nor in the `controller-deployment.yaml`.